src/styles/index.scss

```diff
@@ -1,3 +1,60 @@
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  width: 707px;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    gap: 1px;
+    font-size: 30px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transform: translate(0, -20px);
+    transition-duration: 0.5s;
+  }
```

Comment:
The `transition` property is better to be used instead of `transition-duration` as it allows you to specify which properties you want to add the transition effect to. For example, `transition: background-color 0.5s, transform 0.5s;`